src/components/genre-select.tsx

```diff
@@ -0,0 +1,37 @@
+import React, {useState} from "react";
+
+interface GenreSelectProps {
+    genres: string[];
+    selectedGenre: string;
+    onSelect: (genre: string) => void;
+}
+
+export const GenreSelect: React.FC<GenreSelectProps> = ({genres = [], selectedGenre, onSelect}) => {
+    const [selected, setSelected] = useState(selectedGenre);
```

Comment:
The task is to use the `selectedGenre` prop as "a name of currently selected genre", not just as an initial value. So it seems like the inner state is redundant here.